src/test/java/com/google/api/codegen/testdata/php/php_library.baseline

```diff
@@ -157,11 +170,19 @@ use Google\Tagger\CustomNamespace\V1\LabelerGrpcClient;
  */
 class LibraryServiceGapicClient
 {
+    use GapicClientTrait;
+
+    /**
+     * The name of the service.
+     */
+    const SERVICE_NAME = 'google.example.library.v1.LibraryService';
+
     /**
      * The default address of the service.
      */
     const SERVICE_ADDRESS = 'library-example.googleapis.com';
 
+
```

Comment:
Nit: remove newline (if this is difficult because of snippet lang, then don't worry)

Reply:
SG. I was able to get this guy, but it looks like there are lots of other extraneous newlines in this file as well. We should earmark coming back and cleaning it up a bit.